docs/source/install/install.rst

```diff
@@ -0,0 +1,29 @@
+Install ASReview LAB
+====================
+
+ASReview LAB is supported on Windows and macOS.
```

Comment:
Linus is also supported. Download stats show quite some users!

`macOS` -> `MacOSX`

Reply:
Isn't `Mac OS X` an outdated name? https://en.wikipedia.org/wiki/MacOS